Geometry/TrackerCommonData/data/PhaseI/pixfwdCylinder.xml

```diff
@@ -61,6 +61,14 @@ root volume plus an anchor point for positioning.
  <Constant name="CylindersServiceZ0"   value="0*mm"/>
  <Constant name="CylindersServiceZMin" value="[CylindersServiceZoff]"/>
  <Constant name="CylindersServiceZMax" value="[CylindersOuterLength]"/>
+
+ <Constant name="CarbonFiberSheetNearIPRmin" value="101.0*mm"/>
+ <Constant name="CarbonFiberSheetNearIPRmax" value="168.0*mm"/>
+ <Constant name="CarbonFiberSheetNearIPThickness" value="0.2*mm"/>
+
+ <Constant name="FlexCableDepth" value="0.90*mm"/>
```

Comment:
@Michael-Krohn you are modelling *a lot* of cables here, right? since almost 1mm is plenty and the material has higher density than pure aluminium. Just to make sure we are not off by a factor of ten somewhere.